### DISCUSSION TITLE
How to join discord channel?

### DISCUSSION THREAD

**Discussion by kth-tw** [ConsenSys/quorum#1383] [Q&A] [answered]
If I have some problem during building a quorum network, where is the best place to ask someone? Discord?
The discord link under the website can't be used.
<img width="1414" alt="截圖 2022-04-18 上午10 33 50" src="https://user-images.githubusercontent.com/88102592/163745428-8ce3a666-35aa-4ed0-aff0-c97be8202f45.png">

**Comment by baptiste-b-pegasys** [comment, marked as answer]
Thank you for raising this, I will ask to fix this link.
Meanwhile here is the link I have for the GoQuorum channel : https://discord.com/channels/697535391594446898/747810572937986240

Also the github discussions area is also a good place for asking questions.

**Reply by jlazoff** [reply]
Hi, Thank you for reporting this. We will be fixing this asap. 
Much appreciated!

**Reply by jlazoff** [reply]
The discord link has been updated! Thank you!

**Reply by kth-tw** [reply]
The link below [this page](https://consensys.net/docs/goquorum/en/latest/deploy/install/) also need to be updated

**Reply by jlazoff** [reply]
Looping in @NicolasMassart to update the documentation page. Thank you!

**Reply by NicolasMassart** [reply]
Is this link suitable for new Consensys discord users? I mean it's not an invite link, so does it require that you already are a member of the Consensys server to access this link?
